toolchain/lower/file_context.h

```diff
@@ -157,6 +220,32 @@ class FileContext {
 
   auto BuildVtable(const SemIR::Class& class_info) -> llvm::GlobalVariable*;
 
+  auto AddLoweredSpecificForGeneric(SemIR::GenericId generic_id,
+                                    SemIR::SpecificId specific_id) {
+    llvm::SmallVector<SemIR::SpecificId> existing_specifics;
+    auto ls_entry = lowered_specifics.Insert(generic_id, existing_specifics);
```

Comment:
Would it work to do this instead?

```suggestion
    auto ls_entry = lowered_specifics.Insert(generic_id, {});
```